website/src/pages/initiatives.js

```diff
@@ -1,10 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Layout from '@theme/Layout';
 import styles from './initiatives.module.css';
 import CommunityCard from '../components/CommunityCard';
 import { FixedSizeList as List } from 'react-window';
 import MeetingCard from '../components/PastMeetingCard';
 import { pastMeetings, upcomingMeetings } from '../../websiteData/meetings';
+import { Calendar, dateFnsLocalizer } from 'react-big-calendar';
+import format from 'date-fns/format';
+import parse from 'date-fns/parse';
+import startOfWeek from 'date-fns/startOfWeek';
+import getDay from 'date-fns/getDay';
+import 'react-big-calendar/lib/css/react-big-calendar.css';
+
+const locales = {
+  'en-US': require('date-fns/locale/en-US'),
+};
+const localizer = dateFnsLocalizer({
+  format,
+  parse,
+  startOfWeek,
+  getDay,
+  locales,
+});
+// format for adding a meeting is new Date(year , month-1 , date)
+const events = [
```

Comment:
The events array must have following fields:
```
{
    title: 'Documentation Meeting',
    start: new Date(2021, 11, 2, 16, 30, 0), // should be having hour and minutes along with the time
    end: new Date(2021, 11, 2, 17, 0, 0),
    googleCalendarLink: 'link-to-google-calnedar', // onClick: this should take the user to the Google Calendar of the Moja Global Meetings
  },
  ```
  The events array should be placed `websiteData/meetings.js` and the data can be safely replaced with new ones. The array can be imported in the `initiatives.js` component.